otter/convergence/service.py

```diff
@@ -386,6 +386,17 @@ def delete_divergent_flag(tenant_id, group_id, version):
         yield msg('mark-clean-success')
 
 
+def trigger_convergence(tenant_id, group_id):
+    """
+    Trigger convergence on a scaling group
+    """
+    eff = with_log(mark_divergent(tenant_id, group_id),
+                   tenant_id=tenant_id, scaling_group_id=group_id)
+    return eff.on(success=lambda _: msg("mark-dirty-success"),
```

Comment:
You understood it correct @cyli. I didn't realize that the logs were not bound. Thanks for catching it! Now that I think of it, the bounding may not be required since this function is going to be called by rest or scheduler both of which should bind it tenant, group and trans ids before calling it. Let me do that.